openfermionpyscf/__init__.py

```diff
@@ -17,6 +17,8 @@
 from ._version import __version__
 
 try:
-    from ._run_pyscf import run_pyscf
+    from ._run_pyscf import prepare_pyscf_molecule, run_pyscf
 except ImportError:
     raise Exception("Please install PySCF.")
+
+from ._molecular_data import MolecularData
```

Comment:
please change name of file and name of class. Also, you should do imports alphabetically. Thus, you would normally import ._molecular_data before ._run_pyscf, unless there is a circular dependency.